lib/optimizely.rb

```diff
@@ -810,7 +810,7 @@ def get_feature_variable_for_variation(feature_flag_key, feature_enabled, variat
           end
         else
           @logger.log(Logger::DEBUG,
-                      "Feature '#{feature_flag_key}' for variation '#{variation['key']}' is not enabled. Returning the default variable value '#{variable_value}'.")
+                      "Feature '#{feature_flag_key}' is not enabled for user '#{user_id}'. Returning the default variable value '#{variable_value}'.")
```

Comment:
The changes are incomplete here. 

On line 809 it should say `Variable value is not defined. Returning the default variable value <variable value> for  variable <variable key>`

On line 916 it should say `User <user ID> was not bucketed into experiment or rollout for feature flag <feature flag key>. Returning the default variable value <variable value>`